server/pulp/server/managers/repo/publish.py

```diff
@@ -333,7 +333,7 @@ def auto_distributors(self, repo_id):
         return auto_distributors
 
     @staticmethod
-    def queue_publish(repo_id, distributor_id, overrides=None):
+    def queue_publish(repo_id, distributor_id, overrides=None, scheduled_call_id=None):
```

Comment:
This method returns an `AsyncResult` which should be ignored be ignored as counting as a success because of [this check](https://github.com/pulp/pulp/pull/2013/files#diff-abee42f9c3a1b498c07cb513ca7cb974R411). Shouldn't the schedule_call_id actually be passed along to the actual `publish` task?